packages/accordion/package.json

```diff
@@ -31,17 +31,18 @@
     "publish": "yarn npm publish --tolerate-republish --access public",
     "publish:canary": "yarn npm publish --access public --tag canary"
   },
-  "dependencies": {
-    "@availity/mui-icon": "workspace:*"
-  },
   "devDependencies": {
+    "@availity/mui-button": "workspace:^",
+    "@availity/mui-layout": "workspace:^",
+    "@availity/mui-typography": "workspace:^",
```

Comment:
Does the range begin based on the version when it was switched to a karat? And when we need the range updated do we switch back to a star for one commit and then back to a karat? Or do we know of some other way to update the range without needing two separate commits/releases?